HPXMLtoOpenStudio/resources/hpxml_schematron/EPvalidator.xml

```diff
@@ -1188,6 +1190,15 @@
     </sch:rule>
   </sch:pattern>
 
+  <sch:pattern>
+    <sch:title>[HVACControl]</sch:title>
+    <sch:rule context='/h:HPXML/h:Building/h:BuildingDetails/h:Systems/h:HVAC/h:HVACControl'>
+      <sch:assert role='ERROR' test='count(h:SystemIdentifier) = 1'>Expected 1 element(s) for xpath: SystemIdentifier</sch:assert> <!-- See [HVACControl=Heating] and/or [HVACControl=Cooling] -->
+      <sch:assert role='ERROR' test='count(h:extension/h:OnOffThermostatDeadband) &lt;= 1'>Expected 0 or 1 element(s) for xpath: extension/OnOffThermostatDeadband</sch:assert> <!-- deg F, specify a positive value of deadband between setpoint and cutout temperature to model on/off thermostat behavior. The same value applies to both heating and cooling. -->
```

Comment:
Seems like this should have `Temperature` in the name. As written, I would have though it's a boolean value.